indico/htdocs/sass/partials/_jquery-indico-locationpicker.scss

```diff
@@ -0,0 +1,186 @@
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+.i-location-field {
+    max-width: 400px;
+
+    .wrapper {
+
+        .typeahead-container,
+        .typeahead-field {
+            display: inline-block;
+            position: relative;
+            z-index: 1030 !important;
+            float: left;
+
+            &.venue-container {
+                width: 40%;
+                padding-right: 10px;
+            }
+
+            &.room-container {
+                width: 60%;
+
+                input[type='text'] {
+                    width: 100%;
+                }
+            }
+
+            .typeahead-results {
+                position: absolute;
+                left: 0;
+
+                .typeahead-list {
+                    @include box-shadow(none);
+                    margin: 0;
+                    padding: 0;
+                    overflow: auto;
+                    max-height: 30em;
+                    width: 20em;
+                    position: relative;
+                    border-radius: 0;
+                    border-color: $gray;
+
+                    > li {
+                        background: white;
+                    }
+
+                    &.empty {
+                        border-width: 0px 0px 1px 0px;
+                        border-style: solid;
+                        border-color: $gray;
+
+                        li:first-child {
+                            display: none;
+                        }
+                    }
+                }
+
+                .just-use-option {
+                    display: none;
+                    cursor: pointer;
+                    background-color: white;
+                    text-align: center;
+                    padding: 6px 20px;
+                    border-width: 0px 1px 1px 1px;
+                    border-style: solid;
+                    border-color: $gray;
+                    font-style: italic;
+
+                    &:hover, &.active {
+                        background-color: #ebebeb;  // same as in typeahead
+                    }
+                }
+            }
+
+            &.result {
+                .i-location-input-field {
+                    z-index: 1050 !important;
+
+                    &.typeahead-hint {
+                        z-index: -1 !important;
+                        background-color: white !important;
+                    }
+                }
+
+                .just-use-option {
+                    display: block;
+                }
+
+                z-index: 1060 !important;
+            }
+        }
+
+        input.i-location-input-field {
+            width: 11.5em;
+            background-color: inherit;
+            padding-right: 2em !important;
+
+            &.disabled {
+                @extend %tilted-strips-bg;
+            }
+
+            &.bring-to-front {
+                background: white;
+            }
+
+            &:focus {
```

Comment:
@OmeGak I tried to put blue borders (https://github.com/nop33/indico/blob/devel/indico/htdocs/sass/partials/_jquery-indico-locationpicker.scss#L123), but I need to add an `!important` statement to make it work, didn't notice it before, thanks!
And I also agree that it would be nice for all inputs.